src/reloader.cpp

```diff
@@ -82,10 +82,11 @@ void Reloader::reload(unsigned int pos,
 		try {
 			oldfeed->set_status(DlStatus::DURING_DOWNLOAD);
 			std::shared_ptr<RssFeed> newfeed = parser.parse();
-			if (newfeed->total_item_count() > 0) {
+			if (!newfeed->is_query_feed()) {
```

Comment:
I'd move this wa-a-ay higher, right after the definition of `oldfeed`—this method doesn't know how to populate query feeds, so it makes sense to skip them as early as possible. `reload_all` will take care of populating them at the end.

A comment to that effect would be nice as well.

Reply:
Good idea, done.

Also added the comment.
If you have a better suggestion for [the comment](https://github.com/newsboat/newsboat/pull/746/commits/46547a749b1a05bfcc1a5e1d93bc93c01e7319d3#diff-82d82421c3bacb97d6028178bb3e5df3R65-R66), I'm definitely open to that

Reply:
Thanks! The comment looks good to me. I'll merge once the builds finish.